### PR TITLE
NEO-816: Fixing LeftNavigation console errors and adding one more story

### PR DESCRIPTION
**Pull request by enrique-prado**
Fixed some aria-label related errors, added one more story and other clean up.

### CODE REVIEW

File: src/components/LeftNavigation/TopLinkItem/TopLinkItem.stories.tsx [hunk 1]
```diff
@@ -11,43 +12,34 @@ const handleClick = (e: MouseEvent) => {
   alert(`clicked on the item: ${e.currentTarget.textContent}`);
 };
 export const Default = () => (
-  <div className="neo-leftnav--wrapper">
-    <nav className="neo-leftnav">
-      <ul className="neo-leftnav__nav">
```
Comment (enrique-prado) [inline src/components/LeftNavigation/TopLinkItem/TopLinkItem.stories.tsx:16]: Diff is deceptive. Only replaced these first 3 lines of mark up with a <LeftNavigation> component in line 15.

File: src/components/LeftNavigation/LinkItem/LinkItem.stories.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const handleFocus = () => {
 };
 
 export const LinkItems = () => (
-  <LeftNavigation ariaLabel="Main Nav1" currentUrl="" onSelected={handleClick}>
+  <LeftNavigation aria-label="Main Navigation" currentUrl="" onSelected={handleClick}>
```
Comment (joe-s-avaya) [inline src/components/LeftNavigation/LinkItem/LinkItem.stories.tsx:27]: nit: lint error 
Screen Shot 2022-05-10 at 2 42 26 PM

Comment (joe-s-avaya) [reply src/components/LeftNavigation/LinkItem/LinkItem.stories.tsx:27]: and below

**Review by joe-s-avaya — APPROVED** [review]
some linting issues, but nothing major, looks good!

File: src/components/LeftNavigation/NavCategory/NavCategory.stories.tsx [hunk 3]
```diff
@@ -54,3 +54,33 @@ export const WithIcons: Story<NavCategoryProps> = () => (
     </NavCategory>
   </LeftNavigation>
 );
+
+export const TestApp: Story<NavCategoryProps> = () => (
```
Comment (enrique-prado) [inline src/components/LeftNavigation/NavCategory/NavCategory.stories.tsx:58]: New Story

Comment (yangAtSpoken) [reply src/components/LeftNavigation/NavCategory/NavCategory.stories.tsx:58]: Please test this story in the unit test file.

**Review by yangAtSpoken — APPROVED** [review]

**Pull request merged**